3-0-spring-framework/3-3-0-enable-string-trimming/src/main/java/com/bobocode/StringTrimmingConfiguration.java

```diff
@@ -0,0 +1,11 @@
+package com.bobocode;
+
+import org.springframework.context.annotation.Bean;
+
+public class StringTrimmingConfiguration {
```

Comment:
@Stanislav-Zabramnyi please add javadoc (explain that it's porously not marked with `@Configuration`)